blocks/toc/toc.css

```diff
@@ -16,7 +16,7 @@ main .toc-wrapper {
     margin-right: 0;
 }
 
-main .toc-wrapper.flat {
+main .toc-wrapper.flat:not(.no-bottom-spacing) {
```

Comment:
`:not(.no-..)` is a double negative. How about just calling it `with-bottom-spacing` ? BTW, how about just omitting this style and have all `toc` blocks have a bottom padding uniformly?

Reply:
@JiangLong2019 I know you use the horizontal TOC in at least one place
https://main--sunstar--hlxsites.hlx.live/brands/consumer-health-beauty

I need it for this page
https://www.sunstar.com/about/history/

Your TOC requires more bottom spacing through padding/margins; do you think we could somehow split the difference and make the spacing uniform for both of us

Reply:
@dnbute Sure, I think you can change the block `margin-bottom` to 4 or 5rem. It will be more friendly to your case and hope it won't be a big concern to my page ^ ^

Reply:
@JiangLong2019 I added the changes, please take a look and let me know if it's fine
Thanks!